.travis.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         - opam init --yes --bare
```

Comment:
here's one thing I just thought of: why not enable travis CI for the OCaml bindings in ARM as well?